src/satellitePropagator/physics/AccelerationComponents/DragComponent.h

```diff
@@ -5,15 +5,130 @@
 #pragma once
 #include "satellitePropagator/debris/Debris.h"
 #include "satellitePropagator/physics/Constants.h"
+#include <map>
 
 namespace Acceleration::DragComponent {
-namespace {
-}
+/**
+ * Table with values for density in different altitudes.
+ * Units: altitude[km],  density[km/m^3]
```

Comment:
shouldn't density be kg/m^3 ? :)

Reply:
lol yes. a62977c785766aa302da66f86836df0682f003c4